test/unit/specs/utils/validation/createDefaultValidator.spec.js

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2019 Adobe. All rights reserved.
+Copyright 2020 Adobe. All rights reserved.
```

Comment:
FWIW, I've been told by our legal department that updating the year is unnecessary. That policy may have changed or if your IDE is doing it automatically, that's fine with me as well.

Reply:
I updated my IDE to use 2020 for new files.  I think the diff on GitHub mixed up the old files versus new files.